nautobot/cloud/models.py

```diff
@@ -0,0 +1,38 @@
+from django.db import models
+
+from nautobot.core.constants import CHARFIELD_MAX_LENGTH
+from nautobot.core.models.generics import OrganizationalModel
+from nautobot.extras.utils import extras_features
+
+
+@extras_features(
+    "custom_links",
+    "custom_validators",
+    "dynamic_groups",
+    "export_templates",
+    "graphql",
+    "webhooks",
+)
+class CloudAccount(OrganizationalModel):
```

Comment:
Maybe PrimaryModel? I feel like there could be use cases for adding tags to a CloudAccount.